config/horizon.php

```diff
@@ -2,6 +2,8 @@
 
 return [
 
+    'path' => 'horizon',
```

Comment:
Let's also maybe add a descriptive comment above the setting like the other ones?